libvast/vast/concept/parseable/vast/time.hpp

```diff
@@ -143,21 +144,28 @@ struct ymdhms_parser : vast::parser<ymdhms_parser> {
 
   template <class Iterator, class Attribute>
   bool parse(Iterator& f, const Iterator& l, Attribute& x) const {
+    using namespace parser_literals;
     using namespace std::chrono;
-    auto year = integral_parser<int, 4, 4>{}
-                  .with([](auto x) { return x >= 1900; });
-    auto mon = integral_parser<int, 2, 2>{}
-                 .with([](auto x) { return x >= 1 && x <= 12; });
-    auto day = integral_parser<int, 2, 2>{}
-                 .with([](auto x) { return x >= 1 && x <= 31; });
-    auto hour = integral_parser<int, 2, 2>{}
-                 .with([](auto x) { return x >= 0 && x <= 23; });
-    auto min = integral_parser<int, 2, 2>{}
-                 .with([](auto x) { return x >= 0 && x <= 59; });
-    auto sec = parsers::real_opt_dot
-                 .with([](auto x) { return x >= 0.0 && x <= 60.0; });
+    auto year = integral_parser<int, 4, 4>{}.with(
+      [](auto x) { return x >= 1900; });
+    auto mon = integral_parser<int, 2, 2>{}.with(
+      [](auto x) { return x >= 1 && x <= 12; });
+    auto day = integral_parser<int, 2, 2>{}.with(
+      [](auto x) { return x >= 1 && x <= 31; });
+    auto hour = integral_parser<int, 2, 2>{}.with(
+      [](auto x) { return x >= 0 && x <= 23; });
+    auto min = integral_parser<int, 2, 2>{}.with(
+      [](auto x) { return x >= 0 && x <= 59; });
+    auto sec = parsers::real_opt_dot.with(
+      [](auto x) { return x >= 0.0 && x <= 60.0; });
+    auto time_divider = '+'_p | 'T';
+    // clang-format off
     auto p = year >> '-' >> mon
-        >> ~('-' >> day >> ~('+' >> hour >> ~(':' >> min >> ~(':' >> sec))));
+              >> ~('-' >> day
+                >> ~(time_divider >> hour
+                  >> ~(':' >> min
+                    >> ~(':' >> sec >> ~parsers::chr{'Z'}))));
```

Comment:
`parser::chr` has a `char` as attribute. Don't you want just a literal parser, which has no attribute?